lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -155,6 +167,11 @@ def _prepare_environment(self, environment: Environment, log: Logger) -> bool:
                 parent_logger=log,
             )
 
+        if self.platform_runbook.capture_libvirt_debug_logs:
+            self._enable_libvirt_debug_log()
+
+        self.libvirt_conn = libvirt.open(self.libvirt_conn_str)
```

Comment:
Why was this line moved?

Reply:
If I recall correctly, I needed to place this line in `_initialize` to ensure that it is only opened once for all test suites/cases. Otherwise, the console logger would be unhappy when running multiple tests that include a reboot test.

Reply:
Please check the latest iteration. I moved `host_node` also to `initialize` since it doesn't need to be done "per environment".